### DISCUSSION TITLE
Ubuntu Focal support on launchpad ppa:aziotsdklinux/ppa-azureiot

### DISCUSSION THREAD

**Discussion by florianfischerx** [Azure/azure-iot-sdk-c#1983] [Ideas]
I tried installing the azure-iot-sdk-c package from ppa:aziotsdklinux/ppa-azureiot (https://launchpad.net/~aziotsdklinux/+archive/ubuntu/ppa-azureiot) on Ubuntu Focal (20.04.1).

However, this fails because no Release file  / package exists for this version.
Would it be possible to also publish a package for Focal, since this is the current LTS version?

Many thanks!

**Comment by ewertons** [comment]
Hi @florian-fischer-swarm ,
thanks for the request. We have an ongoing discussion regarding this, and we will post an update soon.

As an alternative please take a look at using our vcpkg port: https://github.com/Azure/azure-iot-sdk-c/blob/master/doc/setting_up_vcpkg.md

**Comment by Timple** [comment]
May I ask what the ongoing discussion is about? Isn't it just supporting the latest LTS or is much more work involved?
